### PR TITLE
Add PSI v2 benchmark

### PR DESCRIPTION
**Pull request by qxzhou1010**
Add PSI v2 benchmark

### CODE REVIEW

**Review by 6fj — APPROVED** [review]
lgtm

File: docs/user_guide/psi_v2_benchmark.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+```
+
+## Prepare config file
+
+We use the config file to specify different PSI protocols and input data.
+
+### sender.config
+```
+{
+         "psi_config": {
```
Comment (rivertalk) [inline docs/user_guide/psi_v2_benchmark.md:37]: Seems indent not aligned

Comment (qxzhou1010) [reply docs/user_guide/psi_v2_benchmark.md]: may be fix, please check again.

Comment (qxzhou1010) [reply docs/user_guide/psi_v2_benchmark.md]: may be fix, please check again. Thanks very much.

**Review by rivertalk — APPROVED** [review]
LGTM

**Pull request merged**